pos_partner_location_abstract/models/res_config_settings.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    auto_create_map_data = fields.Boolean(
```

Comment:
Question : 
This fields looks to be not used (and not available in any view).
Did I missed something ?